lib/api/tests/js_externals.rs

```diff
@@ -46,7 +46,7 @@ mod js {
     #[wasm_bindgen_test]
     fn global_set() {
         let mut store = Store::default();
-        let mut env = FunctionEnv::new(&mut store, ());
+        let _env = FunctionEnv::new(&mut store, ());
```

Comment:
If it's not used, we can just remove this line :)

Reply:
done